merged_prs_per_sprint.rb

```diff
@@ -73,12 +65,17 @@ def title_markdown(pr)
 end
 
 def milestone_prs(milestone, milestone_range, fq_repo_name)
-  prs = if milestone
-          stats.pull_requests(fq_repo_name, :state => "closed", :milestone => milestone[:number])
-          # stats.search_issues(fq_repo_name, milestone)
-        else
-          prs_for_repos_without_milestones(fq_repo_name, milestone_range)
-        end
+  params = {:state => "closed", :sort => 'closed_at', :direction => 'desc'}
+
+  prs =
+    if milestone
+      stats.pull_requests(fq_repo_name, params.merge(:milestone => milestone.number))
+    else
+      since = milestone_range.begin.in_time_zone("US/Pacific").iso8601
+      stats.pull_requests(fq_repo_name, params.merge(:since => since)).select do |pr|
```

Comment:
Does this include PRs that were opened before the sprint start, but merged during the sprint?  I think that's why this wasn't in here.

Reply:
good question, I'm not sure...is there a good example of this? This alone accounted for a majority of the time savings.

Reply:
I don't have an example at the moment, but it's fairly common.

Reply:
I ran this script against the last 3 sprints with and without the since parameter and I get the exact same results.  If it's common, then this seems to account for it.

Reply:
ok, I got an idea...let's merge an old PR...I'm thinking if you can update [this PR](https://github.com/ManageIQ/more_core_extensions/pull/63), then we can get it merged and it will verify.

Reply:
Ok I merged https://github.com/ManageIQ/more_core_extensions/pull/63 and it's not showing up...let me see what's wrong...

Reply:
o_O  It also doesn't show up with the original code on master...investigating

Reply:
ohhhh...there's a hardcoded assumption of a [2 week sprint](https://github.com/ManageIQ/sprint_statistics/blob/master/milestone.rb#L52), but we just so happen to be in the 4 week sprint.  I think I need to bring over my sprint number generator to here 😖 

For now, I'm going to just hardcode the range to see if it picks it up...

Reply:
With the fixed "start date" for the current sprint, the code in this PR *does* pick up that new change, so this is working :tada:

I will make a separate PR to fix the milestone range.